hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java

```diff
@@ -120,6 +120,16 @@ public IndexLabel(String database) {
         public Vertex(String database) {
             super(database, TABLE);
         }
+
+        @Override
+        protected BackendColumnIterator queryById(Session session, Id id) {
+            byte[] value = session.get(this.table(), id.asBytes());
+            if (value == null) {
+                return BackendColumnIterator.empty();
+            }
+            BackendColumn col = BackendColumn.of(id.asBytes(), value);
+            return new BackendColumnIteratorWrapper(col);
+        }
```

Comment:
prefer to add a protected method getById() to super class, and just call super.getById() here, also call it from EdgeTable.queryById().

Reply:
prefer to add a protected method getById() to super class, and just call super.getById() here, also call it from EdgeTable.queryById().